Engine/src/Engine/Rendering/UIRenderer.cpp

```diff
@@ -236,5 +238,10 @@ namespace Engine
 	{
 		m_hierarchyPanel.addEntity(tag, entity.getHandle());
 	}
+
+
+	glm::vec2 UIRenderer::getGameWindowPos() {
+		return m_gamePanel.getPos();
```

Comment:
these are part of another PR right? maybe omit these from here for now, or wait until the other PR is approved since it has changes pending

Reply:
These ones in particular, I don't think were included in the other PR. More an @Hevano question though

Reply:
Ill approve the PR thenbut I do recommend to rename this function to getPosition instead of getPos, just for consistency